<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTTestVs10Class</class>
 <widget class="QMainWindow" name="QTTestVs10Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PathFinder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qttestvs10.qrc">
    <normaloff>:/QTTestVs10/pathfinder.png</normaloff>:/QTTestVs10/pathfinder.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionAddTracker"/>
    <addaction name="separator"/>
    <addaction name="actionLoadPreset"/>
    <addaction name="actionImportCSV"/>
    <addaction name="separator"/>
    <addaction name="actionResultComparer"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAnsicht">
    <property name="title">
     <string>Ansicht</string>
    </property>
    <addaction name="actionNotes"/>
    <addaction name="actionAusgabe"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuAnsicht"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddTracker"/>
   <addaction name="separator"/>
   <addaction name="actionLoadPreset"/>
   <addaction name="actionImportCSV"/>
   <addaction name="separator"/>
   <addaction name="actionNotes"/>
   <addaction name="actionAusgabe"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="qttestvs10.qrc">
     <normaloff>:/QTTestVs10/exit.png</normaloff>:/QTTestVs10/exit.png</iconset>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="toolTip">
    <string>Programm beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAusgabe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qttestvs10.qrc">
     <normaloff>:/QTTestVs10/ausgabe.png</normaloff>:/QTTestVs10/ausgabe.png</iconset>
   </property>
   <property name="text">
    <string>Ausgabe</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionAddTracker">
   <property name="icon">
    <iconset resource="qttestvs10.qrc">
     <normaloff>:/QTTestVs10/plus.png</normaloff>:/QTTestVs10/plus.png</iconset>
   </property>
   <property name="text">
    <string>Tracker hinzufügen</string>
   </property>
   <property name="toolTip">
    <string>Erlaubt das erzeugen eines neuen Tracker-Tasks. Mehrere Tracker können parallel arbeiten.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="qttestvs10.qrc">
     <normaloff>:/QTTestVs10/pathfinder.png</normaloff>:/QTTestVs10/pathfinder.png</iconset>
   </property>
   <property name="text">
    <string>Über PathFinder</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="qttestvs10.qrc">
     <normaloff>:/QTTestVs10/help.png</normaloff>:/QTTestVs10/help.png</iconset>
   </property>
   <property name="text">
    <string>Hilfe zu PathFinder</string>
   </property>
  </action>
  <action name="actionLoadPreset">
   <property name="icon">
    <iconset resource="qttestvs10.qrc">
     <normaloff>:/QTTestVs10/importp.png</normaloff>:/QTTestVs10/importp.png</iconset>
   </property>
   <property name="text">
    <string>Preset laden</string>
   </property>
  </action>
  <action name="actionImportCSV">
   <property name="icon">
    <iconset resource="qttestvs10.qrc">
     <normaloff>:/QTTestVs10/import.png</normaloff>:/QTTestVs10/import.png</iconset>
   </property>
   <property name="text">
    <string>Tracking-Ergebnisse importieren</string>
   </property>
  </action>
  <action name="actionNotes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qttestvs10.qrc">
     <normaloff>:/QTTestVs10/notes.png</normaloff>:/QTTestVs10/notes.png</iconset>
   </property>
   <property name="text">
    <string>Notizen</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qttestvs10.qrc">
     <normaloff>:/QTTestVs10/settings.png</normaloff>:/QTTestVs10/settings.png</iconset>
   </property>
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionResultComparer">
   <property name="icon">
    <iconset resource="qttestvs10.qrc">
     <normaloff>:/QTTestVs10/tracker.png</normaloff>:/QTTestVs10/tracker.png</iconset>
   </property>
   <property name="text">
    <string>Ergebnis-Vergleich</string>
   </property>
   <property name="toolTip">
    <string>Vergleich von Ergebnissen aus verschiedenen Tracking-Vorgängen.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qttestvs10.qrc"/>
 </resources>
 <connections/>
</ui>
